atomic_defi_design/assets/languages/atomic_defi_ru.ts

```diff
@@ -417,7 +417,7 @@
     <message>
         <location filename="../../qml/Wallet/AddressBookSendWalletSelector.qml" line="18"/>
         <source>Choose a valid </source>
-        <translation>Выберите валидную</translation>
+        <translation>Выберите валидную </translation>
```

Comment:
trailing whitespace

Reply:
the problem here is that trailing whitespace is in the English source and this Linguist app for qt/qml demands that the translation should mirror the source.

Reply:
<img width="522" alt="Screenshot 2021-04-27 at 15 12 24" src="https://user-images.githubusercontent.com/11756768/116239156-fdb80680-a76a-11eb-8463-a8b0e7e31104.png">
turns out this is being done on purpose.